debugger/DebuggerThread.hx

```diff
@@ -376,10 +364,10 @@ class DebuggerThread
             }
             // Allow at most 10K of classes in a message
             // Allow at most 100 classes in a message
-            if ((total == 100) || (f.length + byte_total) > (10 * 1024)) {
+            /*if ((total == 100) || (f.length + byte_total) > (10 * 1024)) {
```

Comment:
I didn't know IntelliJ had such problem. It's much better to put the limit back in place then.